ios/AppLovinMAX.m

```diff
@@ -598,9 +608,10 @@ - (instancetype)init
     return @([interstitial isReady]);
 }
 
-RCT_EXPORT_METHOD(showInterstitial:(NSString *)adUnitIdentifier)
+RCT_EXPORT_METHOD(showInterstitial:(NSString *)adUnitIdentifier :(nullable NSString *)placement :(nullable NSString *)customData)
 {
-    [self showInterstitialWithPlacement: adUnitIdentifier : nil];
+    MAInterstitialAd *interstitial = [self retrieveInterstitialForAdUnitIdentifier: adUnitIdentifier];
+    [interstitial showAdForPlacement: placement customData: customData];
 }
 
 RCT_EXPORT_METHOD(showInterstitialWithPlacement:(NSString *)adUnitIdentifier :(NSString *)placement)
```

Comment:
Do we need to keep this? Or can we just use the API that's overloaded with `placement` and `customData`?

Reply:
No, we don't need it anymore.  Let me remove it.